### ISSUE TITLE
UltimateChat hook?

### ISSUE THREAD

**Issue by EmanYm** [nguyenquyhy/DiscordBridge#65]
I see on the Readme.md "todo" list you have custom channel isolation on the todo (future) list.  Will that include things like [UltimateChat](https://ore.spongepowered.org/FabioZumbi12/UltimateChat)?

**Comment by kaal-1** [comment]
Yes please! Discord Bridge once didn't even load when uchat was installed, now it loads but only sends join, leave and death messages to discord. I'm so looking forward to this compatibility
